cf_xarray/accessor.py

```diff
@@ -403,31 +429,33 @@ def get_single(obj: Union[DataArray, Dataset], key: str):
             raise KeyError(f"No results found for {key!r}.")
         return results
 
+    get_single.__doc__ = func.__doc__.replace("One or more of", "One of")
+
     return get_single
 
 
 #: Default mappers for common keys.
 _DEFAULT_KEY_MAPPERS: Mapping[str, Tuple[Mapper, ...]] = {
-    "dim": (_get_dims,),
-    "dims": (_get_dims,),  # transpose
-    "drop_dims": (_get_dims,),  # drop_dims
-    "dimensions": (_get_dims,),  # stack
-    "dims_dict": (_get_dims,),  # swap_dims, rename_dims
-    "shifts": (_get_dims,),  # shift, roll
-    "pad_width": (_get_dims,),  # shift, roll
+    "dim": (_dims(_get_all),),
+    "dims": (_dims(_get_all),),  # transpose
+    "drop_dims": (_dims(_get_all),),  # drop_dims
+    "dimensions": (_dims(_get_all),),  # stack
+    "dims_dict": (_dims(_get_all),),  # swap_dims, rename_dims
+    "shifts": (_dims(_get_all),),  # shift, roll
+    "pad_width": (_dims(_get_all),),  # shift, roll
     "names": (_get_all,),  # set_coords, reset_coords, drop_vars
-    "labels": (_get_indexes,),  # drop_sel
-    "coords": (_get_dims,),  # interp
-    "indexers": (_get_indexes,),  # sel, isel, reindex
+    "labels": (_indexes(_get_all),),  # drop_sel
+    "coords": (_dims(_get_all),),  # interp
+    "indexers": (_dims(_get_all),),  # sel, isel, reindex
     # "indexes": (_get_axis_coord,),  # set_index
-    "dims_or_levels": (_get_dims,),  # reset_index
-    "window": (_get_dims,),  # rolling_exp
-    "coord": (_get_axis_coord_single,),  # differentiate, integrate
+    "dims_or_levels": (_dims(_get_all),),  # reset_index
+    "window": (_dims(_get_all),),  # rolling_exp
+    "coord": (_single(_coords(_get_all)),),  # differentiate, integrate
     "group": (_single(_get_all), _get_groupby_time_accessor),  # groupby
-    "indexer": (_single(_get_indexes),),  # resample
+    "indexer": (_single(_indexes(_get_all)),),  # resample
     "variables": (_get_all,),  # sortby
-    "weights": (_get_measure_variable,),  # type: ignore
-    "chunks": (_get_dims,),  # chunk
+    "weights": (_variables(_single(_get_all)),),  # type: ignore
```

Comment:
I thing we can look at all variables here, right? Before was the equivalent of `(_variables(_single(_get_measure)),),`

Reply:
yes!